sdk/core/Azure.Core/src/Azure.Core.csproj

```diff
@@ -8,7 +8,7 @@
     <PackageTags>Microsoft Azure Client Pipeline</PackageTags>
     <Nullable>enable</Nullable>
     <DefineConstants>$(DefineConstants);AZURE_NULLABLE</DefineConstants>
-    <TargetFrameworks>$(RequiredTargetFrameworks);net461;netcoreapp2.1;net5.0</TargetFrameworks>
+    <TargetFrameworks>$(RequiredTargetFrameworks);net461;netcoreapp3.1;net5.0</TargetFrameworks>
```

Comment:
Won't this be a package breaking change?

Reply:
Yeah, we won't provide 2.1 optimized implementations anymore but ns2.0 target would still be there.

Reply:
Brought the 2.1 target back.